tests/ci/setup_slurm_and_run_tests.sh

```diff
@@ -90,6 +90,21 @@ fi
 
 export SIMPLE_CI=1
 
+sudo --user="${USER}" --login bash -c 'cd ${PWD}/tests; find ./ -name "*.log" -delete'
+
 # Always running at least with '--enable-modules'. No need to check for
 # an empty TESTS array.
 sudo --user="${USER}" --preserve-env=SIMPLE_CI --login bash -c "cd ${PWD}/tests; ./bootstrap; ./configure --disable-all --enable-modules --enable-rms-harness --with-mpi-families='openmpi4 mpich' ${TESTS[*]}; make check"
+
+tests_status=$?
+if [ "${tests_status}" -ne 0 ]; then
```

Comment:
Maybe do a
```shell
if [ "${tests_status}" -eq 0 ]; then
        exit 0
fi
```
here and the log dumping outside of the `if`.

Reply:
Good suggestion! Done!